### PR TITLE
add fonction bundle

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -44,6 +46,7 @@ public function registerBundles()
             new OpenOrchestra\LogBundle\OpenOrchestraLogBundle(),
             new OpenOrchestra\UserAdminBundle\OpenOrchestraUserAdminBundle(),
             new OpenOrchestra\MediaAdminBundle\OpenOrchestraMediaAdminBundle(),
+            new OpenOrchestra\WorkflowAdminBundle\OpenOrchestraWorkflowAdminBundle(),
```
Comment (nicolasThal) [inline app/AppKernel.php]: WorflowAdminBundle and FunctionBundle are not consistent. Can you choose one of them ?

Comment (itkg-ngilain) [inline app/AppKernel.php]: same note

**Comment by itkg-nanne** [comment]
Is it ok, @nicolasThal

File: app/AppKernel.php [hunk 2]
```diff
@@ -37,6 +37,8 @@ public function registerBundles()
             new OpenOrchestra\UserBundle\OpenOrchestraUserBundle(),
             new OpenOrchestra\ModelBundle\OpenOrchestraModelBundle(),
             new OpenOrchestra\MediaBundle\OpenOrchestraMediaBundle(),
+            new OpenOrchestra\WorkflowFonctionBundle\OpenOrchestraWorkflowFonctionBundle(),
+
```
Comment (nicolasThal) [inline app/AppKernel.php]: this empty line is too much I think

**Comment by itkg-nanne** [comment]
renamme done @nicolasThal

**Comment by itkg-ngilain** [comment]
:+1:

**Comment by nicolasThal** [comment]
you also need to commit the composer.lock file

**Pull request merged**